src/TextField/TextField.js

```diff
@@ -108,10 +123,14 @@ export default class TextField extends Component {
         )}
         <Input
           className={inputClassName}
-          value={value}
+          component={multiLine ? 'textarea' : 'input'}
```

Comment:
I have two thoughs here:
1. The `Input` component is public and can be used on his own, hence all the multiline logic should be at the same place, in the `Input`. 
2. Exposing a component property means user can override it. Hence, it should always take precedence, no matter the `multiLine` state.